secrets.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: page
+title: Secrets of Gorilla Repl
+---
+
+Sourced from Lee Spector (lspector@hampshire.edu), revised October 19, 2016
+
+----
+This document provides tips on using the combination of Leiningen (<http://leiningen.org>) and Gorilla REPL (<http://gorilla-repl.org>) **as a complete Clojure programming environment**. The key features for using Gorilla as a complete environment are:
+
+- Open any `.clj`, `.cljs`, `.cljc`, or `.hl` file. If the file does not already have Gorilla REPL worksheet markup, it will appear as a worksheet with the entire contents of the file in a single input cell.
+
+- Save a worksheet without Gorilla REPL worksheet markup. This will save only the text in the worksheet's input areas. Note that this, in conjunction with the first feature listed above, allows you to use Gorilla REPL to open, edit, and save any Clojure file, without introducing unwanted markup.
+
+- Use a leiningen template to easily create a new Clojure project using the `app` template (allowing it to be run from a command line with `lein run`) and also including a dependency to the proper version of the Gorilla REPL plugin.
+
+**To make a project:**
+If you have leiningen installed and you want to create a project called `foo`, for which you can use Gorilla REPL as your development environment, then `cd` into the directory where you want it to live and type:
+
+```
+lein new gorilla-app foo
```

Comment:
I guess we should remember to update the dep in the gorilla-app-template project as well, coordinated with the release.

I wonder whether we ought to add that to the release process docs on the developer page?